packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.tsx

```diff
@@ -34,30 +22,24 @@ export class DetailsRowFields extends BaseComponent<IDetailsRowFieldsProps, IDet
   }
 
   public render(): JSX.Element {
-    const { columns, columnStartIndex, shimmer } = this.props;
+    const { columns, columnStartIndex, shimmer, rowClassNames } = this.props;
     const { cellContent } = this.state;
 
     return (
-      <div
-        className={css('ms-DetailsRow-fields', styles.fields)}
-        data-automationid="DetailsRowFields"
-        role="presentation"
-      >
+      <div className={rowClassNames.fields} data-automationid="DetailsRowFields" role="presentation">
         {columns.map((column, columnIndex) => (
           <div
             key={columnIndex}
             role={column.isRowHeader ? 'rowheader' : 'gridcell'}
             aria-colindex={columnIndex + columnStartIndex + 1}
             className={css(
-              'ms-DetailsRow-cell',
-              styles.cell,
               column.className,
-              column.isMultiline && 'is-multiline',
-              column.isRowHeader && styles.isRowHeader,
-              column.isPadded && styles.isPadded,
-              column.isMultiline && styles.isMultiline,
-              column.isIconOnly && shimmer && styles.shimmerIconPlaceholder,
-              shimmer && styles.shimmer
+              column.isMultiline && rowClassNames.isMultiline,
+              column.isRowHeader && rowClassNames.isRowHeader,
+              column.isPadded && rowClassNames.isPadded,
+              column.isIconOnly && shimmer && rowClassNames.shimmerIconPlaceholder,
+              shimmer && rowClassNames.shimmer,
+              rowClassNames.cell
```

Comment:
I think this ties to @phkuo comment.. will this get worked out such that order is not important once everything is converted? Styling ordering here shouldn't matter, so I'm wondering if shimmer and cell should be mutually exclusive. (I did some work with links in Pivot with exclusion [here](https://github.com/OfficeDev/office-ui-fabric-react/pull/5324/files#diff-4714de560076bfda4e0b0272d915f1adR121)